tests/drivers/gpio/gpio_api_1pin/boards/gpio_pcal6408a.overlay

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I would reduce this to a single `gpio_pcal64xxa.overlay`.

Reply:
obsolete, removed the test